File: server/src/com/broll/gainea/server/core/map/impl/MountainsMap.java
package com.broll.gainea.server.core.map.impl;

import com.broll.gainea.server.core.map.AreaID;
import com.broll.gainea.server.core.map.ContinentID;
import com.broll.gainea.server.core.map.ExpansionFactory;
import com.broll.gainea.server.core.map.ExpansionType;
import com.broll.gainea.server.core.map.IslandID;

public class MountainsMap extends ExpansionFactory {
    public enum Continents implements ContinentID {

    }

    public enum Islands implements IslandID {

    }

    public enum Areas implements AreaID {
    }

    public MountainsMap() {
        super(ExpansionType.MOUNTAINS);
        setBaseCoordinates(-1.05f,1.08f);
    }

    @Override
    public String getTexture() {
        return "expansion_4.png";
    }
    @Override
    protected void init() {

    }

    @Override
    protected void connectWithExpansion(ExpansionFactory expansion) {

    }


}

File: server/src/main/com/broll/gainea/server/core/map/impl/MountainsMap.kt
package com.broll.gainea.server.core.map.impl

import com.broll.gainea.server.core.map.AreaID
import com.broll.gainea.server.core.map.ContinentID
import com.broll.gainea.server.core.map.ExpansionFactory
import com.broll.gainea.server.core.map.ExpansionType
import com.broll.gainea.server.core.map.IslandID

class MountainsMap : ExpansionFactory(ExpansionType.MOUNTAINS, "expansion_3.png") {
    enum class Continents : ContinentID
    enum class Islands : IslandID
    enum class Areas : AreaID

    init {
        baseCoordinates.set(-1.05f, 1.08f)
    }

    override fun init() {}
    override fun connectWithExpansion(expansion: ExpansionFactory) {}
}
